keel-bakery-plugin/src/main/kotlin/com/netflix/spinnaker/keel/bakery/api/ImageSpec.kt

```diff
@@ -12,7 +14,14 @@ data class ImageSpec(
   val storeType: StoreType,
   val artifactStatuses: Set<ArtifactStatus> = enumValues<ArtifactStatus>().toSet(),
   override val application: String // the application an image is baked in
-) : ResourceSpec {
+) : ResourceSpec, UnhappyControl {
   @JsonIgnore
   override val id: String = artifactName
+
+  @JsonIgnore
+  override val maxDiffCount = 2
+
+  // rebaking does not help remedy any of the bake problems we see.
```

Comment:
So zero means "wait forever or until the diff changes". I feel the comment doesn't quite fit with that. Maybe just remove the comment?

Reply:
Sure, I"ll remove it.